integration_tests/suite/test_cdr.py

```diff
@@ -1447,3 +1448,255 @@ def test_list_multitenant(self):
                 has_entries(tenant_uuid=MASTER_TENANT),
             ),
         )
+
+    @call_log(
+        **{'id': 10},
+        tenant_uuid=USERS_TENANT,
+        date='2017-03-23 00:00:00',
+        participants=[{'user_uuid': USER_1_UUID, 'line_id': '1', 'role': 'source'}],
+    )
+    @call_log(
+        **{'id': 11},
+        tenant_uuid=USERS_TENANT,
+        date='2017-03-23 00:00:00',
+        participants=[{'user_uuid': USER_2_UUID, 'line_id': '1', 'role': 'source'}],
+    )
+    @call_log(
+        **{'id': 12},
+        tenant_uuid=OTHER_TENANT,
+        date='2017-03-23 00:00:00',
+        participants=[{'user_uuid': OTHER_USER_UUID, 'line_id': '1', 'role': 'source'}],
+    )
+    @call_log(**{'id': 13}, date='2017-03-23 00:00:00', tenant_uuid=MASTER_TENANT)
+    def test_list_multitenant_token_and_tenant_as_query_string(self):
+        port = self.service_port(9298, 'call-logd')
+
+        response = requests.get(
```

Comment:
Why did you have to use requests here?

Reply:
I wanted to be sure that the client would not add the `Wazo-Tenant` and the `X-Auth-Token` headers to the request.